test/src/integration/integration_test.go

```diff
@@ -167,6 +168,10 @@ func runAnsiblePlaybooks(t *testing.T) {
 	// Install tools
 	runAnsiblePlaybook(t, k8sModuleDir, "../inventories", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../"})
 
+	// Workaround
+	// Because the configuration file assumes that monitoring is enabled
+	runKubectl(t, "create", "ns", "monitoring")
```

Comment:
In this case, we are taking a workaround approach to fix the test.
However, there is also a way to change the config file.

Reply:
Thank you!
Just a question. I noticed we need the `monitoring` namespace from a certain point but is the name specified by the Prometheus operator we use?

I think we can take the following three ways basically?
1. Take the current approach (done in this PR)
1. Disable monitoring
1. Use the default namespace for monitoring for this test (not sure if it is possible)

So, first of all, my understanding is correct?
We should check if monitoring is deployed properly so we should take 1 or 3?
3 is possible?

Reply:
@feeblefakie 

> So, first of all, my understanding is correct?

Yes, I am aware of that. :bow:

> We should check if monitoring is deployed properly so we should take 1 or 3?
> 3 is possible?

I think it is possible to change the namespace of prometheus by modifying this.
https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardl/values.yaml#L77-L96
https://github.com/scalar-labs/scalar-kubernetes/blob/master/playbooks/roles/prometheus/defaults/main.yml#L2